manifests/test_templates.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: test-templates
+  namespace: nixmodules
+  labels:
+    component: 'template-tester'
+    cost-center: 'devex'
+    environment: 'development'
+spec:
+  template:
+    metadata:
+      labels:
+        component: 'template-tester'
+        cost-center: 'devex'
+        environment: 'development'
+    spec:
+      serviceAccountName: default
+      automountServiceAccountToken: true
+      restartPolicy: Never
+      containers:
+        - name: template-tester
+          image: nixos/nix
+          command: ['scripts/test_templates.py', '-m', "{{.Params.nixmodules_version}}"]
+          volumeMounts:
+            - mountPath: /code
+              name: code
+          workingDir: /code/nixmodules
+      initContainers:
+        - args:
+            - "--ssh"
+            - "--repo=git@github.com:replit/nixmodules.git"
+            - "--root=/code"
+            - "--branch=th-tmpl-tests"
```

Comment:
Do we change this in a follow PR or now?

Reply:
Thanks! Changed it to the rev from semaphore instead.